Makefile

```diff
@@ -19,7 +19,8 @@ reprotest: ## Check RPM package reproducibility
 install-deps:
 	sudo dnf install -y \
         git file python3-devel python3-pip python3-qt5 python3-wheel \
```

Comment:
Why are we not installing python3-systemd like we already do for qt5?

Reply:
Fair question - I was thinking of it in terms of having it available in the virtual environment, where it's also useful for the development environment. But it's true that the situation is pretty much the same as with `python-qt5` so going with that approach is better. Will change and rebase